topic_table_model.h

```diff
@@ -184,8 +184,13 @@ class TopicTableModel : public QAbstractTableModel
 
     bool check_rc(DDS::ReturnCode_t rc, const char* what);
 
+    void setDataRow(DataRow* const row, const DDS::DynamicData_var& data, DDS::MemberId id);
```

Comment:
```suggestion
    void setDataRow(DataRow* row, const DDS::DynamicData_var& data, DDS::MemberId id);
```
Leave top-level const (to the right of `&` or `*`) out of function declarations -- it can still be used in definitions